scripts/sync_template_tracker.py

```diff
@@ -22,6 +22,9 @@ def main(trackerbot_url, mark_usable=None):
     unresponsive_providers = set()
     # Queue up list_template calls
     for provider_key in all_providers:
+        ipaddress = cfme_data['management_systems'][provider_key].get('ipaddress', None)
```

Comment:
Won't this prevent us from tracking templates on thins like azure and ec2? as they don't have IP addresses? Though, I don't think we track those any way do we?